R/tm_t_abnormality_by_worst_grade.R

```diff
@@ -502,6 +505,16 @@ srv_t_abnormality_by_worst_grade <- function(id, # nolint
   checkmate::assert_class(shiny::isolate(data()), "teal_data")
 
   shiny::moduleServer(id, function(input, output, session) {
+    isolate({
+      resolved <- teal.transform:::resolve_delayed(worst_flag_indicator, as.list(data()@env))
```

Comment:
Any reason why you're using triple colon here and on the other modules? `resolve_delayed` is an [exported method](https://github.com/insightsengineering/teal.transform/blob/5f116f2d3c84fdcbba91dd376002b8a63876d1ad/NAMESPACE#L61)

```suggestion
      resolved <- teal.transform::resolve_delayed(worst_flag_indicator, as.list(data()@env))
```

Reply:
thanks, I'll change to `::`